frontend/src/services/auth.tsx

```diff
@@ -26,6 +28,30 @@ class AuthService {
     return this.tokenSubject$.pipe(map(token => !!token));
   }
 
+  /**
+   * Gets login of recently registered user
+   */
+  public justRegistered(): Observable<string | null> {
+    return this.registeredUser$.pipe(map(user => user));
```

Comment:
Operator `map(item => item)` is pointless as it doesn't change anything. You can just simply `return this.wantsToLogin$` or make the `wantsToLogin$` subject public